pipelines/cohort_screening/src/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:latest
+
+RUN apt-get update \
+    && apt-get install -y \
+    wkhtmltopdf
+
+RUN apt-get install -y \
+    python3-pip idle3
+
+RUN python3 -m pip install --no-cache-dir --upgrade pip
+
+COPY requirements.txt .
+
+RUN python3 -m pip install -r requirements.txt
+
+COPY . /src
+
+# builds with GitHub Action "Main Validation and Release" ../.github/workflows/build-and-publish-main.yaml
+# tag = us-east4-docker.pkg.dev/broad-gotc-prod/variantreport:${{steps.get-artifact-slug.outputs.slug}}, us-east4-docker.pkg.dev/broad-gotc-prod/variantreport:latest
+
+# For local Dev
+# Google Artifact Registry
+# docker build -t us-east4-docker.pkg.dev/broad-gotc-prod/variantreport:<new_version> .
+# then push, if you want to push
+# Azure Container Registry
```

Comment:
Do we want to move this to DEVELOPMENT.md

Reply:
I like having this in the Dockerfile, but I don't feel super strong about that. We need to have a larger conversation with the whole M42 pipelines team about how we want to organize our repositories, so I'll leave this till then for consensus.